tests/integration/_experimental/test_quality_standard.py

```diff
@@ -164,3 +164,115 @@ def test__download_quality_standard_result(
             ]
             == waterloo_minimum
         )
+
+
+def test__download_quality_standard_result__union(
+    datapoints: pd.DataFrame,
+    results: List[Tuple[EvalConfig, pd.DataFrame]],
+) -> None:
+    dataset_name = with_test_prefix("test__download_quality_standard_result__union")
+    model_name = with_test_prefix("test__download_quality_standard_result__union_model")
+    model_name2 = with_test_prefix("test__download_quality_standard_result__union_model_2")
+    eval_config, model1_results = results[0]
+    upload_dataset(dataset_name, datapoints, id_fields=ID_FIELDS)
+    _upload_results(
+        dataset_name,
+        model_name,
+        model1_results,
+    )
+    model_2_results = model1_results[2:]
+    _upload_results(
+        dataset_name,
+        model_name2,
+        model_2_results,
+    )
```

Comment:
could we normalize the affix style between `model_name`, `model_name2`, `model1_results` and `model_2_results`?